sdk/eventhub/perf-tests/event-hubs-track-1/test/receive.spec.ts

```diff
@@ -82,7 +81,7 @@ async function RunTest(
   for (let partitionId of partitionIds) {
     consumerClient.receive(partitionId, onMessageHandler, onErrorHandler, {
       eventPosition: EventPosition.fromStart(),
-      consumerGroup
+      consumerGroup,
```

Comment:
this will fail our formatting check. Please make to format the code correctly before merging.

Reply:
This is an independent npm project.
And using the latest version of prettier, so our CI couldn't catch this and it would still be valid.

I'll revert to the older prettier to make all the projects use the same version.

Reply:
I see, then please make sure the prettier scripts use our repo-wide [prettier config](https://github.com/Azure/azure-sdk-for-js/blob/main/.prettierrc.json).

Reply:
In case I was not clear, you can keep using the recent version of prettier but you need to make sure it uses the right config file. The issue in this file does not have anything to do with prettier version itself. After doing that, please run `rushx format` to re-format the files.

Reply:
Ran `rushx format`.